screenpipe-app-tauri/lib/hooks/use-settings.tsx

```diff
@@ -322,7 +322,7 @@ export function useSettings() {
           showScreenpipeShortcut: savedShowScreenpipeShortcut,
           isFirstTimeUser: savedIsFirstTimeUser,
           enableFrameCache: savedEnableFrameCache,
-          enableUiMonitoring: false,
+          enableUiMonitoring: true,
```

Comment:
i'd keep it false by default unless we are able to switch from OCR to this easily